sparkle/platform/settings_help.py

```diff
@@ -779,6 +789,22 @@ def get_parallel_portfolio_number_of_seeds_per_solver(self: Settings) -> int:
         return int(
             self.__settings["parallel_portfolio"]["num_seeds_per_solver"])
 
+    def set_run_on(self: Settings, value: Runner = Runner.SLURM,
+                   origin: SettingState = SettingState.DEFAULT) -> None:
+        """Set the compute on which to run."""
+        section = "general"
+        name = "run_on"
+
+        if value is not None and self.__check_setting_state(
+                self.__run_on_set, origin, name):
+            self.__init_section(section)
+            self.__run_on_set = origin
+            self.__settings[section][name] = value
+
+    def get_run_on(self: Settings) -> Runner:
```

Comment:
Does this return an Enum or just a string? Sparkle will probably still work if its a string but it could be nice to actually return the Enum from RunRunner

Reply:
It returns a string and does, indeed, work with the rest of Sparkle. But I agree, it is nicer to return the actual Enum. Just changed it.